scripts/zones/Ilrusi_Atoll/IDs.lua

```diff
@@ -18,6 +18,7 @@ zones[xi.zone.ILRUSI_ATOLL] =
         LOGIN_CAMPAIGN_UNDERWAY       = 7002, -- The [/January/February/March/April/May/June/July/August/September/October/November/December] <number> Login Campaign is currently underway!
         LOGIN_NUMBER                  = 7003, -- In celebration of your most recent login (login no. <number>), we have provided you with <number> points! You currently have a total of <number> points.
         MEMBERS_LEVELS_ARE_RESTRICTED = 7023, -- Your party is unable to participate because certain members' levels are restricted.
+        ASSAULT_START_OFFSET          = 7354, -- Max MP Down removed for <name>.
```

Comment:
I think this is 7454?

Reply:
It's a bit different from assault zone to assault zone. This was working when I developed this assault.